components/finance/FinanceDateHeader/index.tsx

```diff
@@ -52,23 +49,38 @@ const useStyles = makeStyles(() =>
 
 const isEqual = (a: any, b: any): Boolean => a <= b && a >= b
 
-export default function FinanceDateHeader({ startDate, endDate, activeColumn }: any) {
-  const dates = []
+const getHeadings = () => gql`
+  query Finance {
```

Comment:
This query should accept start and end date. As of now mock data is sending a specific set of data but real data will be huge if start and end dates are not specified